src/miral/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ add_library(miral SHARED
     cursor_theme.cpp                    ${miral_include}/miral/cursor_theme.h
     display_configuration.cpp           ${miral_include}/miral/display_configuration.h
     external_client.cpp                 ${miral_include}/miral/external_client.h
+    fd_manager.cpp                      ${miral_include}/miral/fd_manager.h
```

Comment:
`fd_manager.cpp` is not for use by client code, it should be built as part of `miral-internal` (which ensures that no entry points are leaked)